internal/test/upload_test.go

```diff
@@ -454,6 +464,8 @@ func verifyFilesWereIsolated(t *testing.T, fileIDs []string) {
 		prodMatches, _ := fs.Glob(fsys, filepath.Join(prod, fmt.Sprintf("%s.*", fileIDs[i])))
 
 		total := len(betaMatches) + len(prodMatches)
-		require.Greater(t, total, 0, fmt.Sprintf("fileID[%d] %s not found in beta or prod shard", i, fileIDs[i]))
+		if total == 0 {
+			t.Errorf("fileID[%d] %s not found in beta or prod shard", i, fileIDs[i])
```

Comment:
Why did you decide to move away from the require package for this check?

Reply:
`require.NoError` stops the test's execution so we only are shown the first instance of missing files. `t.Error` continues the test (but still fails it at the end). `t.Fatal` would stop the test after the first failure.

Reply:
That makes sense. Thank you for explaining.